prompts/awesome-chatgpt-prompts_agent/scripts/conv.js

```
@@ -2,7 +2,7 @@ const dataSet = require("./prompt.json");
 const fs = require("fs");
 
 const trim = (message) => {
-  return message.replace(/^"/, "").replace(/"$/, "").replace(/\(SLP\)/, "").replace("'", "")
+  return message.replace(/^"/, "").replace(/"$/, "").replace(/\(SLP\)/, "").replace(/'/g, "")
 };
```

ESLint

[error] Missing semicolon.

(semi)